src/components/manage/Blocks/Listing/SummaryTemplate.jsx

```diff
@@ -27,7 +27,9 @@ const SummaryTemplate = ({ items, linkTitle, linkHref, isEditMode }) => {
             <ConditionalLink item={item} condition={!isEditMode}>
               <Component componentName="PreviewImage" item={item} alt="" />
               <div className="listing-body">
-                <h3>{item.title ? item.title : item.id}</h3>
+                <ul className="listing-item-ul">
+                  <li>{item.title ? item.title : item.id}</li>
+                </ul>
```

Comment:
this doesn't work.

There should be just one `<ul></ul>` and each item rendered should be a `<li></li>`

Here we have a <ul> per item.

The <ul> should go above the `.items.map((item)` and everything inside it should be a `<li>`

Reply:
Moreover should we use plain <ul> <li>-s or should we use [Semantic UI react Lists](https://react.semantic-ui.com/elements/list/)?

this way we can even have a `<List.Header>` with the title and so on.

We can even use the [Component Registry](https://6.docs.plone.org/volto/configuration/component-registry.html) register the List component there, and then the customization is way easier.

Reply:
thank you for the feedback, I have applied changes using Semantic UI components